sqlalchemy_bigquery/__init__.py

```diff
@@ -41,6 +41,9 @@
     BIGNUMERIC,
 )
 
+# Define a "dialect" attribute to follow the common SqlAlchemy convention assumed by some libraries that leverage SqlAlchemy.
+base.dialect = dialect = BigQueryDialect
```

Comment:
If you'd like me to write a test, please let me know what file I should add the test to.

Reply:
Thanks! I'd put this in `tests/unit/test_sqlalchemy_bigquery.py`.